pkg/tests/ossm/yaml_configs.go

```diff
@@ -35,10 +38,11 @@ var (
 
 func GetSMCPTemplates() map[string]string {
 	return map[string]string{
-		"2.4": smcpV24_template,
-		"2.3": smcpV23_template,
-		"2.2": smcpV22_template,
-		"2.1": smcpV21_template,
+		"2.4":         smcpV24_template,
+		"2.3":         smcpV23_template,
+		"2.3-minimal": smcpV23_template_minimal,
```

Comment:
I added this template, because I don't want to waste time on installing all addons when I need to test cert-manager.